Islandora-Devops/isle-alpaca
# Initial Alpaca image

Creates an initial image containing Karaf and all alpaca services.

## Known issues:
* The image is named `birkland/isle-alpaca` for the time being, until a dockerhub repo is chosen for the purposes of ISLE
* There is no automatic push to dockerhub
* There are no tests

## To Test
* `docker-compose build`, then `docker-compose up -d`
* Go to `http://localhost:8161`, which is the ActiveMQ admin console.  Click on "Manage ActiveMQ broker".  Enter in username `admin`, pass `admin`
* Click on the `queues` tab.  You should see all the queues the connectors listen to. 
* Now, pick a service (maybe fits) and remember it.  Shut everything down `docker-compose down -v`
* To to the `.env` file and comment out the env vars for that service.  This will disable it.
* Start up `docker-compose up -d`
* log into ActiveMQ.  Look at the queues, and observe the one that corresponds to your service doesn't exist.  This proves you can shut them off at will
